ignore pycache in sha1 calculation (#3474)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased

> * Fixed an issue in calculating content graph hash when running **graph create** and **graph update** commands.

Reviewer: ```suggestion
* Fixed an issue in calculating content graph hash when creating or updating it.
```

> * Fixed an issue where running **graph create** and **graph update** commands without arguments invoked `help` command.

Reviewer: ```suggestion
* Calling **graph create** or **graph update** now run the commands with default arguments, instead of showing the command help.
```
